src/appleseed/renderer/modeling/aov/depthaov.cpp

```diff
@@ -128,18 +132,18 @@ namespace
 
         size_t get_channel_count() const override
         {
-            return 1;
+            return 2;
         }
 
         const char** get_channel_names() const override
         {
-            static const char* ChannelNames[] = { "Z" };
+            static const char* ChannelNames[] = { "Z", "A" };
             return ChannelNames;
         }
 
         void clear_image() override
         {
-            m_image->clear(Color<float, 1>(numeric_limits<float>::max()));
+            m_image->clear(Color<float, 1>(0.0f));
```

Comment:
You need to clear with a two-channel color here.

Reply:
`Color<float, 2>(0.0f)` should work.

Reply:
what should be the default values for each of them?